source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,133 @@
+- Copy files from the old platform to the Pantheon Drupal 9 Site
+- Create a Git-tracked local copy of the Pantheon Drupal 9 Site
+- Export the database and files from the old platform, but do not add them or upload any files to Pantheon.
+
+Return to this doc once you have created a full local copy of the Pantheon Drupal 9 Site and have the database and files ready.
+
+### Site Structure
+
+<Partial file="ic-upstream-structure.md" />
+
+## Migrate the Old Drupal Site's Code to the Pantheon Drupal 9 Site
+
+1. Copy over exported configuration from the original site. From the Pantheon D9 site, run the following commands:
+
+  ```bash{promptUser: user}
+  mkdir config
+  git mv sites/default/config/* config/
+  git commit -m "Add site configuration."
+  ```
+
+1. List the contrib modules and themes on the old site:
+
+  ```bash{promptUser: user}
+  composer show
+  ```
+
+1. Use Composer to add each module and theme on the Pantheon Drupal 9 site:
+
+  ```bash{promptUser: user}
+  composer require drupal/ctools:^3.4 drupal/redirect:^1.6 drupal/token:^1.7
+  git add composer.*
+  git commit -m "Add contrib projects."
+  ```
+
+1. Use Composer to install the requirements on the Pantheon Drupal 9 site, then push the changes to the platform:
+
+  ```bash{promptUser: user}
+  composer install
+  git add . && git commit -m "composer install" && git push origin master
+  ```
+
+1. Copy any custom modules or themes from the old site to the Pantheon site:
+
+  ```bash{promptUser: user}
+  mkdir web/{themes, modules}
+  git mv themes/* web/themes
+  git mv modules/* web/modules
+  git commit -m "Add custom projects."
+  ```
+
+1. Check `settings.php` for any customizations to copy over:
+
+  ```bash{promptUser: user}
+  mkdir -p web/sites/default
+  git show sites/default/settings.php > web/sites/default/original-settings.php
+  # Check for any customizations (if this returns nothing, you can move on to the next step).
+  # Copy what you need over to web/sites/default/settings.php, and commit as needed.
+  diff -Nup web/sites/default/settings.php web/sites/default/original-settings.php
+  # Remove the original copy.
+  rm web/sites/default/original-settings.php
+  ```
+
+1. Use Terminus to import the old site's database (created in the [migrate manual](/migrate-manual#add-your-database) doc) into the Pantheon D9 site. This example uses a local `db.sql.gz` file. If your DB archive file is located at a URL, replace the file name with the full URL in this example:
+
+  ```bash{promptUser: user}
+  terminus import:database $SITE.dev ~/db.sql.gz
+  ```
+
+1. Use Terminus to import the old site's files (created in the [migrate manual](/migrate-manual#upload-your-files) doc) into the Pantheon D9 site. This example uses a local `files.tar.gz` file. If your DB archive file is located at a URL, replace the file name with the full URL in this example:
+
+  ```bash{promptUser: user}
+  terminus import:files $SITE.dev ~/files.tar.gz
```

Comment:
`terminus import:files` will only use a publicly-accessible URL.. unfortunately can't be used with a local file like this.